### PR TITLE
[KOGITO-1727] - Release Kogito Images 0.9.1

### PR DESCRIPTION
**Pull request by spolti**
staging data-index: data-index-service-0.9.1-runner.jar
md5: 805fed350dbeb03b699e1bf99374d64f

staging jobs-service: jobs-service-0.9.1-runner.jar
md5: 72cf21f32a3777f59beaf0994026ee59

staging management-console: management-console-0.9.1-runner.jar
md5: ceb538a8ef0a30efd26a422cab9f5f94

images available on quay.io under tag 0.9.1-rc3:
- quay.io/kiegroup/kogito-data-index:0.9.1-rc3
- quay.io/kiegroup/kogito-jobs-service:0.9.1-rc3
- quay.io/kiegroup/kogito-management-console:0.9.1-rc3
- quay.io/kiegroup/kogito-quarkus-jvm-ubi8:0.9.1-rc3
- quay.io/kiegroup/kogito-quarkus-ubi8:0.9.1-rc3
- quay.io/kiegroup/kogito-quarkus-ubi8-s2i:0.9.1-rc3
- quay.io/kiegroup/kogito-springboot-ubi8:0.9.1-rc3
- quay.io/kiegroup/kogito-springboot-ubi8-s2i:0.9.1-rc3

Signed-off-by: spolti <fspolti@redhat.com>

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

File: kogito-quarkus-s2i-overrides.yaml [hunk 1]
```diff
@@ -51,3 +51,5 @@ packages:
   - glibc-devel
   - zlib-devel
 
+run:
```
Comment (radtriste) [inline kogito-quarkus-s2i-overrides.yaml:54]: This is fix in https://github.com/kiegroup/kogito-images/pull/142 or ?

File: modules/kogito-data-index/module.yaml [hunk 2]
```diff
@@ -4,8 +4,9 @@ version: "0.9.1-rc2"
 
 artifacts:
   - name: kogito-data-index-runner.jar
-    url: https://repository.jboss.org/org/kie/kogito/data-index-service/0.9.1-SNAPSHOT/data-index-service-0.9.1-20200410.154951-11-runner.jar
-    md5: cf5549ad15725a3a4e69f085da56683b
+    #url: https://repository.jboss.org/org/kie/kogito/data-index-service/0.9.1-SNAPSHOT/data-index-service-0.9.1-20200410.154951-11-runner.jar
+    url: https://repository.jboss.org/nexus/content/repositories/jboss_releases_staging_profile-16865/org/kie/kogito/data-index-service/0.9.1/data-index-service-0.9.1-runner.jar
```
Comment (radtriste) [inline modules/kogito-data-index/module.yaml]: `jboss_releases_staging_profile-16865` 
do we want this for final version in repository ?

File: modules/kogito-management-console/module.yaml [hunk 3]
```diff
@@ -8,8 +8,9 @@ envs:
 
 artifacts:
   - name: kogito-management-console-runner.jar
-    url: https://repository.jboss.org/org/kie/kogito/management-console/0.9.1-SNAPSHOT/management-console-0.9.1-20200410.155354-11-runner.jar
-    md5: afd4885e4c33884e744adaaed3e280b7
+    #url: https://repository.jboss.org/org/kie/kogito/management-console/0.9.1-SNAPSHOT/management-console-0.9.1-20200410.155354-11-runner.jar
+    url: https://repository.jboss.org/nexus/content/repositories/jboss_releases_staging_profile-16865/org/kie/kogito/management-console/0.9.1/management-console-0.9.1-runner.jar
```
Comment (radtriste) [inline modules/kogito-management-console/module.yaml]: same here for `jboss_releases_staging_profile-16865`

File: modules/kogito-jobs-service/module.yaml [hunk 4]
```diff
@@ -4,8 +4,9 @@ version: "0.9.1-rc2"
 
 artifacts:
   - name: kogito-jobs-service-runner.jar
-    url: https://repository.jboss.org/org/kie/kogito/jobs-service/0.9.1-SNAPSHOT/jobs-service-0.9.1-20200410.154450-11-runner.jar
-    md5: d7e781cbdb4b5968185699e03614a437
+    #url: https://repository.jboss.org/org/kie/kogito/jobs-service/0.9.1-SNAPSHOT/jobs-service-0.9.1-20200410.154450-11-runner.jar
+    url: https://repository.jboss.org/nexus/content/repositories/jboss_releases_staging_profile-16865/org/kie/kogito/jobs-service/0.9.1/jobs-service-0.9.1-runner.jar
```
Comment (radtriste) [inline modules/kogito-jobs-service/module.yaml]: same here for `jboss_releases_staging_profile-16865`

**Review by radtriste — COMMENTED** [review]
See my comments below

**Comment by radtriste** [comment]
Are we changing also the version to 0.9.1 in this PR ?

**Comment by spolti** [comment]
New images:

staging data-index: data-index-service-0.9.1-runner.jar
md5: 3f0967d34abd163c25e88ffde45c860d

staging jobs-service: jobs-service-0.9.1-runner.jar
md5: 5045a15e32c2d50ef2b2a05ee241de7e

staging management-console: management-console-0.9.1-runner.jar
md5: 6ccb41e412ab9df7cd463ee9e9360b6d

images available on quay.io under tag 0.9.1-rc3:

- quay.io/kiegroup/kogito-data-index:0.9.1-rc3
- quay.io/kiegroup/kogito-jobs-service:0.9.1-rc3
- quay.io/kiegroup/kogito-management-console:0.9.1-rc3
- quay.io/kiegroup/kogito-quarkus-jvm-ubi8:0.9.1-rc3
- quay.io/kiegroup/kogito-quarkus-ubi8:0.9.1-rc3
- quay.io/kiegroup/kogito-quarkus-ubi8-s2i:0.9.1-rc3
- quay.io/kiegroup/kogito-springboot-ubi8:0.9.1-rc3
- quay.io/kiegroup/kogito-springboot-ubi8-s2i:0.9.1-rc3

File: modules/kogito-data-index/module.yaml (hunk 2 above)
Comment (spolti) [reply modules/kogito-data-index/module.yaml]: no, just temporary, it still as draft.

File: modules/kogito-jobs-service/module.yaml (hunk 4 above)
Comment (spolti) [reply modules/kogito-jobs-service/module.yaml]: see comment above.

File: kogito-quarkus-s2i-overrides.yaml (hunk 1 above)
Comment (spolti) [reply kogito-quarkus-s2i-overrides.yaml:54]: > Are we changing also the version to 0.9.1 in this PR ?

yes.

File: modules/kogito-management-console/module.yaml (hunk 3 above)
Comment (spolti) [reply modules/kogito-management-console/module.yaml]: see previous comment

**Review by radtriste — APPROVED** [review]

**Comment by radtriste** [comment]
BDD tests ok with those new images

**Review by ricardozanini — APPROVED** [review]

**Review by radtriste — APPROVED** [review]
Lgtm

**Comment by spolti** [comment]
Latest tests:

```
2020-04-20 17:30:37,101 cekit        INFO     Finished!
2020-04-20 17:30:37,101 - cekit - INFO - Finished!
```

**Pull request merged**